.github/workflows/linear-pr-links.yml

```diff
@@ -0,0 +1,104 @@
+name: Update PR Description with Linear Link
+
+on:
+  push:
+    branches:
+      - '*'  # Triggers on push to any branch that has an open PR
+
+jobs:
+  update-pr-description:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Find Linear issues and update PR description
+        uses: actions/github-script@v6
+        env:
+          LINEAR_API_KEY: ${{ secrets.LINEAR_API_KEY }}
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const branchName = context.payload.ref.replace('refs/heads/', '');
+
+            // Get PRs associated with this branch
```

Comment:
I think you can do it the other way around, get the branchName from the PR event?